src/util/internalCallTree.js

```diff
@@ -189,6 +194,24 @@ function includeVariableDeclaration (tree, step, sourceLocation, scopeId) {
       }
     })
   }
+  var functionDefinition = resolveFunctionDefinition(tree, step, previousSourceLocation)
+  if (functionDefinition && newLocation && traceHelper.isJumpDestInstruction(tree.traceManager.trace[step - 1])) {
+    // means: the previous location was a function definition && JUMPDEST
+    // => we are at the beginning of the function and input/output are setup
+    tree.solidityProxy.contractNameAt(step, (error, contractName) => { // cached
+      if (!error) {
+        tree.traceManager.getStackAt(step, (error, stack) => {
+          if (!error) {
+            var states = tree.solidityProxy.extractStatesDefinitions()
+            // input params
+            addParams(functionDefinition.children[0], tree, scopeId, states, contractName, previousSourceLocation, stack.length, functionDefinition.children[0].children.length, -1)
+            // output params
+            addParams(functionDefinition.children[1], tree, scopeId, states, contractName, previousSourceLocation, stack.length, 0, 1)
```

Comment:
The output params are not stored in reverse.

Reply:
that the reason of the last param I think